go.mod

```diff
@@ -1,21 +1,21 @@
 module github.com/xakep666/mongo-migrate
 
+go 1.21.3
```

Comment:
I ran "go mod tidy". Then the go version was written to `go.mod`.
I realize this is common behavior.